include/kmer.hpp

```diff
@@ -87,6 +87,7 @@ struct uint_kmer_t {
 
     static constexpr uint16_t uint_kmer_bits = 8 * sizeof(Kmer);
     static constexpr uint8_t bits_per_char = BitsPerChar;
+
     // max *odd* size that can be packed into uint_kmer_bits bits
     static constexpr uint16_t max_k = []() {
```

Comment:
It might make more sense to put `uint_kmer_bits / bits_per_char` here, and overload it to use odd `k` for DNA only.